Library/ViewModels/SurveyResponseViewModel.swift

```diff
@@ -166,3 +170,7 @@ private func isSurvey(request: URLRequest) -> Bool {
   guard case (.project(_, .survey, _))? = Navigation.match(request) else { return false }
   return true
 }
+
+private func isStripeElement(_ request: URLRequest) -> Bool {
+  return request.url?.host == "js.stripe.com"
```

Comment:
Can/should we qualify this any more specifically, or is just the host sufficient? I'm not a security engineer but is there any risk to blanket allowing all requests to Stripe's domain?

Reply:
Good question! Confirmed with web ppl that this is fine: https://kickstarter.slack.com/archives/C06EKGN4MHU/p1719343241132189?thread_ts=1719337090.551589&cid=C06EKGN4MHU

Reply:
Awesome, thank you for checking!